spring-orm/src/test/java/org/springframework/orm/jpa/JpaTransactionManagerTests.java

```diff
@@ -424,7 +425,7 @@ public void testPropagationSupportsAndRequiresNewAndEarlyAccess() {
 		assertThat(!TransactionSynchronizationManager.isSynchronizationActive()).isTrue();
 
 		Object result = tt.execute(status -> {
-			EntityManagerFactoryUtils.getTransactionalEntityManager(factory);
```

Comment:
PROPAGATION_SUPPORTS says, that if there is no existing transaction, the code will be executed non-transactionally, and, thus it makes sense for get _Transactional_ EntityManager to return null.